source/routes.ts

```diff
@@ -185,11 +184,16 @@ export class RouteManager {
     return (req, res, next) => {
       const { path, method } = req;
       const routePath = path.replace(options.basePath || '', '');
-      const route = this.findRouteByPath(routePath);
-      const routeMethod = this.findRouteMethod(routePath, method);
 
-      res.locals.route = route;
-      res.locals.routeMethod = routeMethod;
+      try {
+        const route = this.findRouteByPath(routePath);
+        const routeMethod = this.findRouteMethod(routePath, method);
+
+        res.locals.route = route;
```

Comment:
In the previous behavior, this was throwing an error instead of handling gracefully.